docs/chapters/subcommands/bootstrap.rst

```diff
@@ -2,22 +2,37 @@
 bootstrap
 =========
 
-The bootstrap sub-command is used to download and extract releases and
-templates for use with Bastille containers. A valid release is needed before
-containers can be created. Templates are optional but are managed in the same
-manner.
+The `bootstrap` sub-command is used to download and extract releases and
+templates for use with Bastille containers.   First, some background. 
 
-Note: your mileage may vary with unsupported releases and releases newer
-than the host system likely will NOT work at all. Bastille tries to filter for
-valid release names. If you find it will not bootstrap a valid release, please
-let us know.
+Background
+============
 
-In this document we will describe using the `bootstrap` sub-command with both
-releases and templates. We begin with releases.
+A release is a version of the operating system used in jails and shared by one or more Bastille containers. It is mounted in a container as a non-writable  (nullfs) file system. 
```

Comment:
Please wrap all lines at 80 characters (as it already is in the rest of the document)